omero/sysadmins/admins-with-restricted-privileges.txt

```diff
@@ -0,0 +1,418 @@
+:term:`Chgrp`                                     N                     N                      N                       Y
+:term:`Chown`                                     N                     N                      Y (O)                   Y
+:term:`Create and Edit Groups`                    N                     N                      N                       Y
+:term:`Create and Edit Users`                     N                     N                      N                       Y
+:term:`Add Users to Groups`                       N                     N                      N                       Y
+:term:`Upload Scripts`                            N                     N                      Y                       N
+
+=============================== ======================= ======================= ===================== ================================
+
+Y == privilege required, checkbox in OMERO.web interface is checked
+N == privilege not required, checkbox is not checked
+(O) == this privilege is suggested as optional for the workflow
+
+.. note::
+    **Restricted admins workflows in OMERO.clients:** Please do not
+    expect for any workflows mentioned here that all OMERO.clients
+    OMERO.web, OMERO.insight, command line interface (CLI)) are fully
+    equipped to execute them (see details below). New features will
+    be added in OMERO.clients in the 5.3.x series of OMERO releases.
+
+.. note::
+    **Group allegiances** All the workflows here do not assume that
+    the administrator with restricted privileges is not a member of
+    any group except the System group. This does not preclude such
+    administrator from being a member of any number of groups.
+    Inside the groups the restricted admin is a member of, he/she
+    has the same privileges as other group members of that group
+    additionally to his/her adminstrative privileges.
+
+.. note::
+    **Deleting privileges**: :term:`Sudo` privilege has implicitly
+    in itself a Delete power over the data the administrator is
+    sudoed in as. If you want to prevent restricted admin from
+    Deleting of others' data entirely, do not give :term:`Delete Data`
+    AND do not give :term:`Sudo` privileges.
+
+.. note::
+    **Privilege escalation**: Administrator with restricted privileges
+    (restricted admin) are prevented from escalation of their privileges.
+    Escalation cannot be achieved by them neither by sudoing as full
+    administrator or higher-privileged restricted admin, nor by creating
+    a full administraotor or new restricted admin with privileges higher
+    than the creating restricted admin. A restricted admin can use
+    their :term:`Sudo` privilege, the prevention of escalation means
+    they will be able to execute maximally the privileges level they
+    were enjoying before sudoing, no more privileges can be acquired
+    by the sudoing itself. In case the sudoed-as user has less privileges
```

Comment:
As a layman I find this whole note to be confusing.  Maybe a table could detail the privilege escalation better?